README.md

```diff
@@ -41,10 +41,10 @@ _Note: All input types will be checked for existence and checksum matching._
 
 | File Type | Description |
 | :-------: | ------ |
-| bam | Validate bam/cram/sam using `pysam`. <br> Check for an index file in same directory as the BAM.<br><br>_Note: If a BAM input is missing an accompanying BAM index file in the same directory,<br> `validate` will not throw an exception but will print a warning._|
-| vcf | Validate vcf using `vcftools` |
-| fasta |  |
-| bed | |
+| BAM | Validate bam/cram/sam using `pysam`. <br> Check for an index file in same directory as the BAM.<br><br>_Note: If a BAM input is missing an accompanying BAM index file in the same directory,<br> `validate` will not throw an exception but will print a warning._|
+| VCF | Validate vcf using `vcftools` |
+| FASTA |  |
+| BED | |
 | py | |
```

Comment:
It would be nice to make this column consistent. Maybe we could either use extensions like `.bam`, `.vcf`, etc or maybe py -> "Python script" or something?

Reply:
Makes sense, I changed it to `Python script`